src/cli.ts

```diff
@@ -1,9 +1,9 @@
 /* eslint-disable no-process-exit */
 
-import { readdir, readFile, stat, writeFile } from 'fs';
-import { basename, dirname, extname, join } from 'path';
-import { convert, modernizeJS } from './index';
-import { Options } from './options';
+import {readdir, readFile, stat, writeFile} from 'mz/fs';
+import {basename, dirname, extname, join} from 'path';
+import {convert, modernizeJS} from './index';
+import {Options} from './options';
```

Comment:
In a separate PR we should enforce the spacing convention via tslint. In the meantime, can you change these back and match the existing style?